### ISSUE TITLE
Challenge for dns-01

### ISSUE THREAD

**Issue by philkunz** [anatolsommer/letiny#6]
Can I actually use the challenge for dns-01 with letsencrypt?

**Comment by anatolsommer** [comment]
No, sorry, http-01 only. Maybe I'll have time to look into this during christmas holiday.

**Comment by philkunz** [comment]
That would be awesome as letiny is exactly the low level kind of plugin I'm looking for without much of the fuzz of the letsencrypt module on npm.

**Comment by philidem** [comment]
I'm also interested in using `letiny` for `dns-01`. I used the `letsencrypt` module on NPM and got it working but it was way too complicated, poorly documented, and used very odd conventions. I'd like to help with the `dns-01` challenge implementation so let me know if you have any thoughts on that.

**Comment by philkunz** [comment]
@philidem exactly
I started working on a smartacme class that uses le-acme-core, however it keeps running into errors for requesting an account. So as a matter of time I started working on something else for now. If you want take a look here: https://gitlab.com/pushrocks/smartacme/blob/master/ts/smartacme.classes.smartacme.ts

**Comment by philkunz** [comment]
@anatolsommer 
Happy New Year! Any chance you could take a look at this?

**Comment by anatolsommer** [comment]
Sorry, not yet since I had to fight a flood of water from my kitchen ceiling and the repair work still lasts.
I've still got more than a week before I have to go back to work so probably will release a beta version later this week.

**Comment by anatolsommer** [comment]
Sorry, took a while but it should be working now.
(Not yet released on NPM since I want to do some more live tests within the next days)

**Comment by anatolsommer** [comment]
Sorry, just realized I didn't release this version on npm yet. Planned for this weekend.
